bundles/com.espressif.idf.terminal.connector.serial/src/com/espressif/idf/terminal/connector/serial/launcher/CoreDumpPostmortemDebuggerLauncher.java

```diff
@@ -0,0 +1,272 @@
+/*******************************************************************************
+ * Copyright 2022 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.terminal.connector.serial.launcher;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.eclipse.core.resources.IFile;
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.runtime.IPath;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Path;
+import org.eclipse.debug.internal.core.LaunchConfiguration;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import com.espressif.idf.core.IDFCorePlugin;
+import com.espressif.idf.core.IDFEnvironmentVariables;
+import com.espressif.idf.core.ProcessBuilderFactory;
+import com.espressif.idf.core.logging.Logger;
+import com.espressif.idf.core.util.IDFUtil;
+import com.espressif.idf.core.util.SDKConfigJsonReader;
+import com.espressif.idf.ui.IDFConsole;
+
+/**
+ * Core dump debugging configuration creator and launcher. We are using simply the postmortem c/c++ debug configuration
+ * style to create a launch configuration xml file and then simply launch that through eclipse.
+ *
+ * @author Ali Azam Rana
+ *
+ */
+@SuppressWarnings("restriction")
+public class CoreDumpPostmortemDebuggerLauncher implements ISerialWebSocketEventLauncher
+{
+	private static final String CORE_DUMP_POSTMORTEM_LAUNCH_CONFIG = "core_dump_postmortem_debug.launch"; //$NON-NLS-1$
```

Comment:
Could we generate this file name dynamically based on the project name? Something like 'core_dump_projectName.launch'. This would be useful especially if you are working with multiple projects and enabled core dump debugging for all of them

Reply:
@alirana01 not sure if this is addressed?